src/main/java/uk/gov/ons/census/casesvc/utility/FieldworkHelper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ons.census.casesvc.utility;
+
+import uk.gov.ons.census.casesvc.model.entity.Case;
+
+public class FieldworkHelper {
+  public static boolean shouldSendCaseToField(Case caze, String eventChannel) {
+    if (eventChannel.equals("FIELD")) {
```

Comment:
Quick question - Will this event come from Field or solely from a bulk file? Or is it to safeguard in case?

Reply:
we made the re-usable class for some future work that's planned - it's not strictly needed for this specific PR